controls/cis_sle15.yml

```diff
@@ -625,7 +625,7 @@ controls:
     status: automated
     rules:
       - service_rpcbind_disabled
-      - package_rcpbind_removed
+      - package_rpcbind_removed
```

Comment:
@rumch-se this is still present, please remove it from this PR and send it in another PR.

Reply:
Hello @dodys

I can remove package_rpcbind_removed, but its points to the rule
/linux_os/guide/services/nfs_and_rpc/disabling_nfs/disabling_nfs_services/package_rpcbind_removed

or ... I have to remove something else ?
Please describe

Have a nice day
Rumen